### PR TITLE
feat: appwrite dpa

### PR DESCRIPTION
**Pull request by ArmanNik**

### CODE REVIEW

File: src/routes/console/organization-[organization]/settings/downloadDPA.svelte [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+</script>
+
+<CardGrid>
+    <div>
+        <Heading tag="h6" size="7">Download DPA document</Heading>
+    </div>
+    <p class="text">
+        After downloading, have the DPA signed by your organization's compliance authority (e.g.,
+        CEO or Compliance Manager) and submit it to privacy@appwrite.io.
```
Comment (gewenyu99) [inline src/routes/console/organization-[organization]/settings/downloadDPA.svelte:29]:
```suggestion
        After downloading, have the DPA signed by your organization's compliance authority, such as your CEO or Compliance Manager, and submit it to privacy@appwrite.io.
```

Comment (gewenyu99) [reply src/routes/console/organization-[organization]/settings/downloadDPA.svelte:29]: I would go as far as to omit the examples for CEO and compliance manager. I think it's for the organization to decide who this person should/can be.

Can we also make sure the email's a link?

File: src/routes/console/organization-[organization]/settings/downloadDPA.svelte [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    </p>
+    <svelte:fragment slot="aside">
+        <Box>
+            <h6>
+                <b>DPA document (Data Processing Agreement)</b>
```
Comment (gewenyu99) [inline src/routes/console/organization-[organization]/settings/downloadDPA.svelte:34]:
```suggestion
                <b>Data Processing Agreement (DPA) document</b>
```

Comment (gewenyu99) [reply src/routes/console/organization-[organization]/settings/downloadDPA.svelte:34]: Grammar.

Comment (gewenyu99) [reply src/routes/console/organization-[organization]/settings/downloadDPA.svelte:34]: There will be docs under `/docs/advanced/security/dpa`

File: src/routes/console/organization-[organization]/settings/downloadDPA.svelte [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            </h6>
+            <p class="text u-margin-block-start-8">
+                Legal document outlining the terms and conditions under which personal data is
+                processed.
```
Comment (gewenyu99) [inline src/routes/console/organization-[organization]/settings/downloadDPA.svelte:38]: Let's settle for something simpler and link to docs under `/docs/advanced/security/dpa`

Maybe:

The DPA is a legal document that describes the roles and responsibilities of Appwrite and the organization when personal data is processed. Learn more about the DPA.

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: src/routes/console/organization-[organization]/settings/downloadDPA.svelte [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        link.click();
+    }
+</script>
+
+<CardGrid>
+    <div>
+        <Heading tag="h6" size="7">Download DPA document</Heading>
+    </div>
+    <p class="text">
+        After downloading, have the DPA signed by your organization's compliance authority, such as your CEO or Compliance Manager, and submit it to privacy@appwrite.io.
```
Comment (gewenyu99) [inline src/routes/console/organization-[organization]/settings/downloadDPA.svelte:31]: Should the email be a link?

File: src/routes/console/organization-[organization]/settings/downloadDPA.svelte [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    </p>
+    <svelte:fragment slot="aside">
+        <Box>
+            <h6>
+                <b>Data Processing Agreement (DPA) document</b>
+            </h6>
+            <p class="text u-margin-block-start-8">
+              The DPA is a legal document that describes the roles and responsibilities of Appwrite and the organization when personal data is processed. Learn more about the DPA.
```
Comment (gewenyu99) [inline src/routes/console/organization-[organization]/settings/downloadDPA.svelte:39]:
```suggestion
              The DPA is a legal document that describes the roles and responsibilities of Appwrite and the organization when personal data is processed. Learn more about the DPA.
```

The learn more about DPA should be a link 👀

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

File: src/routes/console/organization-[organization]/settings/downloadDPA.svelte (hunk 5 above)
Comment (gewenyu99) [reply src/routes/console/organization-[organization]/settings/downloadDPA.svelte:39]: I'll provide a link when the docs are finalized.

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**